lib/y2lan_restart_common.pm

```diff
@@ -150,12 +150,14 @@ sub check_network_status {
     assert_script_run 'cat journal.log';                              # print journal.log
     my $is_empty = script_run('[ -s journal.log ]');
     if ($is_empty) {
-        assert_equals($expected_status, 'no_restart', "Network restart didn't occur, even though it was expected.");
+        assert_equals($expected_status, 'no_restart', "Do not expect Network restart/reload. journalctl cannot find events");
     }
     else {
-        assert_equals($expected_status, 'restart', "Network restart occurred, even though it wasn't expected.");
+        my $journal_findings = script_output('cat journal.log');
+        my $expected_status  = is_sle('15-SP2+') ? 'reload' : 'restart';
+        assert_equals($expected_status, $expected_status, "Expect Network restart/reload. Journalctl found \n$journal_findings\n\n");
```

Comment:
something wrong here , this line does not make sense , no matter what  `$expected_status` will be equal to `$expected_status`

Reply:
fixed according to the other comments.